data/assets/css/components/_event-summary.scss

```diff
@@ -77,13 +98,22 @@
     text-align: left;
     color: $text-color-secondary;
     font-size: 0.9rem;
+    margin-left: $thumbnailSizeMobile;
```

Comment:
I hope that with flex/grid instead of float this margin-left is not needed. But if it is truly needed, could you move it to https://github.com/VLCTechHub/VLCTechHub-site/blob/master/data/assets/css/abstract/_variables.scss in kebab-case?

Reply:
I'll take a look as soon as the float issue is resolved